R-package/R/utils.R

```diff
@@ -308,18 +308,60 @@ xgb.createFolds <- function(y, k = 10)
 #' @name xgboost-deprecated
 NULL
 
-#' Do not use saveRDS() for long-term archival of models. Use xgb.save() instead.
+#' Do not use \code{\link[base]{saveRDS}} or \code{\link[base]{save}} for long-term archival of
+#' models. Instead, use \code{\link{xgb.save}} or \code{\link{xgb.save.raw}}.
 #'
-#' It is a common practice to use the built-in \code{saveRDS()} function to persist R objects to
-#' the disk. While \code{xgb.Booster} objects can be persisted with \code{saveRDS()} as well, it
-#' is not advisable to use it if the model is to be accessed in the future. If you train a model
-#' with the current version of XGBoost and persist it with \code{saveRDS()}, the model is not
-#' guaranteed to be accessible in later releases of XGBoost. To ensure that your model can be
-#' accessed in future releases of XGBoost, use \code{xgb.save()} instead. For more details and
-#' explanation, consult the page
+#' It is a common practice to use the built-in \code{\link[base]{saveRDS}} function (or
+#' \code{\link[base]{save}}) to persist R objects to the disk. While it is possible to persist
+#' \code{xgb.Booster} objects using \code{\link[base]{saveRDS}}, it is not advisable to do so if
+#' the model is to be accessed in the future. If you train a model with the current version of
+#' XGBoost and persist it with \code{\link[base]{saveRDS}}, the model is not guaranteed to be
+#' accessible in later releases of XGBoost. To ensure that your model can be accessed in future
+#' releases of XGBoost, use \code{\link{xgb.save}} or \code{\link{xgb.save.raw}} instead.
+#'
+#' @details
+#' Use \code{\link{xgb.save}} to save the XGBoost model as a stand-alone file. You may opt into
+#' the JSON format by specifying the JSON extension. To read the model back, use
+#' \code{\link{xgb.load}}.
+#'
+#' Use \code{\link{xgb.save.raw}} to save the XGBoost model as a sequence (vector) of raw bytes
+#' in a future-proof manner. Future releases of XGBoost will be able to read the raw bytes and
+#' re-construct the corresponding model. To read the model back, use \code{\link{xgb.load.raw}}.
+#' The \code{\link{xgb.save.raw}} function is useful if you'd like to persist the XGBoost model
+#' as part of another R object.
+#'
+#' For more details and explanation about model persistence and archival, consult the page
 #' \url{https://xgboost.readthedocs.io/en/latest/tutorials/saving_model.html}.
```

Comment:
Here is the link.

Reply:
Do you want me to edit saving_model.rst ?

Reply:
Yes.

Reply:
Done.